src/js/activity-block.js

```diff
@@ -120,21 +120,23 @@ const checkIsTaskCorrect = (regexForScoreAndPoints) => {
 };
 
 const activityBlock = () => {
-  if (window.location.href.includes("https://todoist.com/app/activity")) {
-    connectFonts();
+  if (!window.location.href.includes("https://todoist.com/app/activity")) {
+    return;
+  }
+  connectFonts();
 
-    const sectionElem = document.getElementsByClassName("section");
-    const items = document.querySelectorAll("ul.items");
-    const itemsArray = Array.from(items);
-    const regexForScoreAndPoints = /^.*\[(?<score>\d+)\]\s*.*$/;
+  const sectionElem = document.getElementsByClassName("section");
```

Comment:
Use full names, as abbreviations tend to be confusing

```suggestion
  const sectionElement = document.getElementsByClassName("section");
```